google-appengine-standard/README.md

```diff
@@ -0,0 +1,38 @@
+# google-appengine-standard
+
+Sample project for [Ktor](http://ktor.io) running under [Google Appengine](https://cloud.google.com/appengine/)
```

Comment:
Google App Engine

Reply:
Fixed. Also added more detailed description of deployment steps.